src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ParseNodeMetricsTests.java

```diff
@@ -37,13 +37,14 @@
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.metrics.PerformanceAnalyzerMetrics;
 
 public class ParseNodeMetricsTests extends AbstractReaderTests {
+    long currentTimeMills = System.currentTimeMillis();
     public ParseNodeMetricsTests() throws SQLException, ClassNotFoundException {
         super();
     }
 
     @Test
     public void testParseMasterPendingMetrics() throws Exception {
-        long currTimestamp = System.currentTimeMillis() + 4000;
+        long currTimestamp = System.currentTimeMillis() + 94000;
```

Comment:
why do we want to not hardcode timestamps in these tests as well?

Reply:
Makes sense; will do in the next rev